lib/fs.js

```diff
@@ -1260,8 +1260,8 @@ fs.writeFile = function(path, data, options, callback_) {
   });
 
   function writeFd(fd, isUserFd) {
-    var buffer = (data instanceof Buffer) ? data : new Buffer('' + data,
-        options.encoding || 'utf8');
+    var buffer = (data instanceof Buffer) ?
+      data : Buffer.from('' + data, options.encoding || 'utf8');
```

Comment:
it's preferred that statement continuations are double indented. this should be fixed in linter.